data/tsi.src

```diff
@@ -92343,6 +92343,7 @@
 狗尾續貂 0 ㄍㄡˇ ㄨㄟˇ ㄒㄩˋ ㄉㄧㄠ
 狗屁 26 ㄍㄡˇ ㄆㄧˋ
 狗屁不通 5 ㄍㄡˇ ㄆㄧˋ ㄅㄨˋ ㄊㄨㄥ
+狗屁倒灶 10 ㄍㄡˇ ㄆㄧˋ ㄉㄠˇ ㄗㄠˋ
```

Comment:
I thought this is what I've done and record in the commit message, did I miss or misunderstand anything here?